cki-send.py

```diff
@@ -75,11 +83,17 @@ def main(args):
                     result_file_name = f'{instance_type}-{test}.result'
                     result_url = f"{args.build_url}artifact/results/{result_file_name}"
 
+                    # this will fail if the test took longer than a day because ansible's duration is not machine readable
+                    # see https://groups.google.com/g/ansible-devel/c/7C1cLrAS9BU
+                    total_seconds = (datetime.strptime(duration, "%H:%M:%S.%f") - datetime.strptime("", "")).total_seconds()
```

Comment:
Not the first time I need to do something like this, but I chose to use regexes as it allows handling the "days" case.

```python
In [34]: cases = [
    ...:     "2 days, 1:02:04",
    ...:     "1 day, 1:02:04",
    ...:     "1:02:04",
    ...: ]

In [35]: for time in cases:
    ...:     days, hours, minutes, seconds = re.match(r'(?:(\d+) day[s]?, )?(\d+):(\d+):(\d+)', time).groups()
    ...:     time = int(days or 0) * 86400 + int(hours) * 3600 + int(minutes) * 60 + int(seconds)
    ...:     print(time)
    ...: 
176524
90124
3724
```

Not sure if this helps?